packages/trackerless-network/test/end-to-end/random-graph-with-real-connections.test.ts

```diff
@@ -15,11 +15,11 @@ describe('random graph with real connections', () => {
     }
 
     const streamPartId = StreamPartIDUtils.parse('random-graph#0')
-    let epDhtNode: DhtNode
-    let dhtNode1: DhtNode
-    let dhtNode2: DhtNode
-    let dhtNode3: DhtNode
-    let dhtNode4: DhtNode
+    let epDhtNode: DhtNode  // TODO is this Layer0 or Layer1?
```

Comment:
DhtNodes here are practically Layer0 nodes acting as Layer1 nodes, for the purpose of this test they are Layer1 nodes as the Dht is per stream.